tests/lcs/core/logic/isstrictly.livecodescript

```diff
@@ -33,36 +33,45 @@ private command __TestIsStrictly pValue, pExpected, pExpectResult
    end repeat
 end __TestIsStrictly
 
-function __GenerateNothing
-   local tNothing
-   put param(1) into tNothing
-   return tNothing
-end __GenerateNothing
-
 on TestStrictlyNothing
-  -- FIXME can't currently generate nothing values in LCS
-  -- __TestIsStrictly __GenerateNothing(), "nothing"
+   __TestIsStrictly nothing, "nothing"
+
+   local tVar
+   __TestIsStrictly tVar, "nothing"
```

Comment:
Is it worth adding a test for unpassed parameters too? (Which I believe should also be `nothing`). e.g.
```
on Foo pA, pB, pC
  test pB is strictly nothing
end Foo

on callFoo
  foo 1, , 2
end callFoo
```
Note: They might not be currently, but probably should be...

Reply:
We should probably also check what param(N) returns where N > paramCount as that is the same case as an unpassed argument.

Reply:
Good idea

Reply:
So having quickly checked the engine source (MCParameter::eval_ctxt; and EngineEvalParam) it looks like unspecified parameters are kMCEmptyString. I don't think it would hurt to change them to kMCNull though - it would bring consistency with uninitialized variables and has no backwards-compatibility issues beyond the existing uses of 'is strictly' (which changing boolean constants has touched on anyway).

Reply:
CommandArgumentsAtIndex should probably be the same